src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -198,6 +198,7 @@ public function getListIdsAction(int $productId): JsonResponse
 
     /**
      * @AdminSecurity("is_granted('delete', request.get('_legacy_controller'))")
+     * @todo: rename to deleteAction
```

Comment:
You better do it now, since this PR already introduces a BC break because of the route name changes

Reply:
yep that was inteded for this PR, forgotten, thanks :v: